### PR TITLE
Add a English version of contributing_cn.md

### PR DESCRIPTION
**Pull request by DingQK**
Resolve: #763 
Signed-off-by: DingQK <18210394728@163.com>

### CODE REVIEW

**Comment by LocoRichard** [comment]
Hi @DingQK , thank you so much for your interest in contributing to PyMilvus Project. I have reviewed your translation. Unfortunately, I found it failed to meet our criteria for documentation. We cannot accept unedited machine translation. However, we really appreciate your effort. Stay tuned if you are still interested in contributing to this project.

**Comment by DingQK** [comment]
> Hi @DingQK , thank you so much for your interest in contributing to PyMilvus Project. I have reviewed your translation. Unfortunately, I found it failed to meet our criteria for documentation. We cannot accept unedited machine translation. However, we really appreciate your effort. Stay tuned if you are still interested in contributing to this project.

Hi! This is not a machine translation. I translate it myself and check grammar mistakes with grammarly (an app for grammar mistakes checking).

**Comment by DingQK** [comment]
> Hi @DingQK , thank you so much for your interest in contributing to PyMilvus Project. I have reviewed your translation. Unfortunately, I found it failed to meet our criteria for documentation. We cannot accept unedited machine translation. However, we really appreciate your effort. Stay tuned if you are still interested in contributing to this project.

Hi @LocoRichard ! This is not a machine translation. I translate it myself and check grammar mistakes with grammarly (an app for grammar mistakes checking). If the content is wrong, I can change the content according to your need. Thank you!

**Comment by LocoRichard** [comment]
> > Hi @DingQK , thank you so much for your interest in contributing to PyMilvus Project. I have reviewed your translation. Unfortunately, I found it failed to meet our criteria for documentation. We cannot accept unedited machine translation. However, we really appreciate your effort. Stay tuned if you are still interested in contributing to this project.
> 
> Hi @LocoRichard ! This is not a machine translation. I translate it myself and check grammar mistakes with grammarly (an app for grammar mistakes checking). If the content is wrong, I can change the content according to your need. Thank you!

I am sorry for the misunderstanding. The content is so far correct. The only problem is that some of the translation does not conform to the style of English writing.
If that is the case, would you mind polishing the language together with me? It will take a bit longer for me to comment and you to make changes accordingly.

**Comment by DingQK** [comment]
> > > Hi @DingQK , thank you so much for your interest in contributing to PyMilvus Project. I have reviewed your translation. Unfortunately, I found it failed to meet our criteria for documentation. We cannot accept unedited machine translation. However, we really appreciate your effort. Stay tuned if you are still interested in contributing to this project.
> > 
> > 
> > Hi @LocoRichard ! This is not a machine translation. I translate it myself and check grammar mistakes with grammarly (an app for grammar mistakes checking). If the content is wrong, I can change the content according to your need. Thank you!
> 
> I am sorry for the misunderstanding. The content is so far correct. The only problem is that some of the translation does not conform to the style of English writing. If that is the case, would you mind polishing the language together with me? It will take a bit longer for me to comment and you to make changes accordingly.

Yes! I can revise according to your need, and I believe I can learn a lot in the process. Thank you so much！

**Comment by LocoRichard** [comment]
> 

Brilliant! I will let you know when I finish reviewing and commenting.

**Comment by DingQK** [comment]
> Brilliant! I will let you know when I finish reviewing and commenting. And a huge THANK for your interest and patience.

Ok!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,45 +1,93 @@
...
-You may need a milvus server which is running when you run unit test. See more details on [Milvus server](https://github.com/milvus-io/milvus).
+If you are interested in existing PRs, you can:
+- join the code review and offer advice
+- guide newcomers to finish the PR process
 
+Note: the problems, features, and questions mentioned here are not limited to Python code. They also refer to all kinds of documents (technical documents, API references, contributing guide, etc.)
 
-## Update CHANGLOG.md
+## PyMilvus Code Structure
+`docs/`: Contains source codes for documentation except for API Reference and uses sphinx to create documentation. Most documentation are in folder `rst` under `docs/source`.
```
Comment (LocoRichard) [inline CONTRIBUTING.md:49]: `docs/`: Contains source documentation (except for API Reference) that created by sphinx; most documentation are stored as `.rst` files under `docs/source`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,45 +1,93 @@
...
 
-Add issue tips into CHANGLOG.md, make sure all issue tips are sorted by issue number in ascending order.
+`examples/`: Contains python scripts, which can be run directly, to introduce the usage of PyMiluvs API through examples.
```
Comment (LocoRichard) [inline CONTRIBUTING.md:51]: "Python"

Use upper case whenever needed.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,45 +1,93 @@
...
 
+`pymilvus/`：PyMilvus source codes.
```
Comment (LocoRichard) [inline CONTRIBUTING.md:53]: Contains

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,45 +1,93 @@
...
 
+`tests/`: Unit Test.
```
Comment (LocoRichard) [inline CONTRIBUTING.md:55]: Contains

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,45 +1,93 @@
...
 
+`CONTRIBUTING.md`: Contributing guidelines.
+
+`CONTRIBUTING_CN.md`: Contributing guidelines in Chinese.
+
+`LICENSE`: Open Source License that PyMilvus follows.
+
+`Makefile`: Scripts for Github action.
+
+`OWNERS`: This file designates reviewers and approvers for the current directory. They are chosen according to their activity and code contribution. Active contributors are listed as reviewers, undertaking the responsibility of code reviews. Reviewers who have been active and reviewing codes for a period of time are listed as approvers. They are in charge of reviewing content apart from codes. If you create a PR and do not know who to review, you can assign reviewers and approvers from this file to review your PR.
```
Comment (LocoRichard) [inline CONTRIBUTING.md:65]: `OWNERS`: This file designates reviewers and approvers for the current directory. They are chosen according to their participation and code contribution. Active contributors are listed as reviewers, responsible for code reviews. Reviewers who have been active and reviewing codes for a period of time are listed as approvers. They are in charge of reviewing content apart from codes. If you submit a PR and do not know who can help you review the code, you can assign reviewers and approvers from this file to review your PR.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,45 +1,93 @@
...
+
+### What will reviewers focus on in reviewing PR?
+
+### What will approvers focus on in reviewing PR?
+
+### Pass all Github Actions
+
+## Congratulations! You are now the contributor to the Milvus community!
+
+Apart from dealing with codes and machines, you can always communicate with people from the Milvus community. Many new faces join in every day, and if you have encountered the same challenges as they are now facing, feel free to help them. You can pass on the collaborative spirit from the helps you acquired when you first join the community. Let us build a collaborative, open-source, open, and tolerant community together!
```
Comment (LocoRichard) [inline CONTRIBUTING.md:93]: Apart from dealing with codes and machines, you are always welcome to communicate with any member from the Milvus community. New faces join us every day, and they may as well encounter the same challenges as you faced beore. Feel free to help them. You can pass on the collaborative spirit from the assistance you acquired when you first joined the community. Let us build a collaborative, open-source, exuberant, and tolerant community together!

**Comment by LocoRichard** [comment]
> > Brilliant! I will let you know when I finish reviewing and commenting. And a huge THANK for your interest and patience.
> 
> Ok!

HI, I have gone through the doc and made some comments, please check and give me some feedback if necessary.

**Review by LocoRichard — CHANGES_REQUESTED** [review]

**Comment by DingQK** [comment]
Hi @LocoRichard ! Thank you so much for your comments and revisal! I have learned a lot about how to clearly convey ideas in a formal and concise way. Right now, I am wondering do we need to remove all the h2 and h3 without content or we can simply leave them there?

I will revise according to the comments and commit as soon as possible. Thank you so much!

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,45 +1,90 @@
...
+- It is a pure Python project;
+- It supports Milvus E2E, Benchmark, and Milvus Bootcamp;
+- It supports significantly more application scenarios as a Python package.
 
+We are committed to building a collaborative, exuberant open-source community for PyMilvus. Therefore, contributions to PyMilvus are welcome from everyone. Anyone who is familiar with the code and usage of PyMilvus is welcome to contribute to the community, help newcomers, and pass on the open-source, collaborative, and open spirit.
 
-## How can I contribute?
 
-### Development environment
+# What contributions can you make?
```
Comment (LocoRichard) [inline CONTRIBUTING.md]: This is a h2.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,45 +1,90 @@
...
+- Review the code and offer advice;
+- Instruct new contributors to complete the PR process.
 
-```shell 
-$ pytest --cov=milvus/client --cov-report=html
-```
+Note: the problems, features, and questions mentioned here are not limited to Python code. They also refer to all kinds of documents (technical documents, API references, contributing guide, etc.)
 
-You may need a milvus server which is running when you run unit test. See more details on [Milvus server](https://github.com/milvus-io/milvus).
+# PyMilvus Code Structure
```
Comment (LocoRichard) [inline CONTRIBUTING.md]: h2 too

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -1,45 +1,90 @@
...
+
+# Merge PR
+
+### What will reviewers focus on in reviewing PR?
+
+### What will approvers focus on in reviewing PR?
+
+### Pass all Github Actions
+
+# Congratulations! You are now the contributor to the Milvus community!
```
Comment (LocoRichard) [inline CONTRIBUTING.md]: h2

**Comment by LocoRichard** [comment]
> Hi @LocoRichard ! Thank you so much for your comments and revisal! I have learned a lot about how to clearly convey ideas in a formal and concise way. Right now, I am wondering do we need to remove all the h2 and h3 without content or we can simply leave them there?
> 
> I will revise according to the comments and commit as soon as possible. Thank you so much!

Yes, we'd better delete those titles without content. Please go ahead. Also, please note, in most scenarios, there is only one h1, which is the title of the article at the top.

We sincerely thank you for your contribution, and the patience all the time!

**Comment by DingQK** [comment]
@LocoRichard  I totally agree. I have deleted the titles and change the titles except the first one into h2 in my latest commit.

**Review by LocoRichard — APPROVED** [review]

**Pull request merged**